content/applications/hr/referrals.rst

```diff
@@ -68,32 +68,34 @@ Onboarding slides can be modified if desired. Only users with :guilabel:`Adminis
 the Recruitment application can modify onboarding slides. To edit a slide, navigate to
```

Comment:
```suggestion
the *Recruitment* application can modify onboarding slides. To edit a slide, navigate to
```

Reply:
Not sure if the sentence: "To edit a slide, navigate to the Recruitment application can modify onboarding slides."
Can you please rework this?

Reply:
This is the full text in this section:

To edit a slide, navigate to :menuselection:`Referrals app --> Configuration --> Onboarding.` Each line displays the text for the individual onboarding slide. To edit an onboarding slide, click on an individual slide line to open the slide's onboarding form.


I don't see what is missing?